packages/lesswrong/server/emailComponents/EventUpdatedEmail.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { getSiteUrl, registerComponent } from '../../lib/vulcan-lib';
+import { useSingle } from '../../lib/crud/withSingle';
+import { postGetPageUrl, prettyEventDateTimes } from '../../lib/collections/posts/helpers';
+
+const styles = (theme: ThemeType): JssStyles => ({
+  root: {
+    fontFamily: "Arial, sans-serif",
```

Comment:
See note about serifed fonts. Also you might consider having sans be an EA-Forum-only choice, as LW's aesthetic is more serifed.

Reply:
Good point about LW, I'll adjust that.

I originally tried to stick with sans-serif for the email heading and serif for the rest, but I personally found the serif harder to read so made this all sans-serif. But I agree that the footer sticks out and I don't want to deal with having to update all our emails here, so to make this less complicated I'll go back to my original plan.